include/seqan/gff_io/gff_io_base.h

```diff
@@ -395,26 +395,29 @@ void readRecord(GffRecord & record, CharString & buffer, TFwdIterator & iter)
     if (buffer != ".")
         record.score = lexicalCast<float>(buffer);
     skipOne(iter, IsTab());
-
+    
     // read column 7: strand
     readOne(record.strand, iter, OrFunctor<OrFunctor<EqualsChar<'-'>, EqualsChar<'+'> >, EqualsChar<'.'> >());
     skipOne(iter, IsTab());
-
+    
     // read column 8: phase
     readOne(record.phase, iter, OrFunctor<EqualsChar<'.'>, IsInRange<'0', '2'> >());
-
+    
     // It's fine if there are no attributes and the line ends here.
     if (atEnd(iter) || isNewline(value(iter)))
     {
         skipLine(iter);
         return;
     }
     skipOne(iter, IsTab());
-
+    // There is often a space character between phase and attribute columns.
+    // We can safely skip that!
+    skipUntil(iter, NotFunctor<IsSpace>());  //skip empty lines
```

Comment:
`Possibly, that some files have tabs instead of spaces.
`
Not possible. Tab is the delimiter in such a file. having two consecutive tabs would mean we have 10 columns instead of 9.